api/model.php

```diff
@@ -0,0 +1,69 @@
+<?php
+  require_once('util.php');
+
+  /*
+    API documentation: https://github.com/keymanapp/keyman/wiki/downloads.keyman.com-model-API
+
+    Gets the available download files for a given model id
+  */
+
+  header('Content-Type: application/json; charset=utf-8');
+  header('Cache-Control: max-age=0');
+
+  $allowed_platforms = array('android', 'ios' /*, 'linux', 'mac', 'web', 'windows'*/); // models only currently supported on iOS and Android
+  $allowed_versions = array('1.0'); // api versions, not product versions
+  $release_tiers = array('alpha', 'beta', 'stable');
```

Comment:
Are `$allowed_platforms` and `$release_tiers` for future use? I see they came from keyboard.php